test/Card-test.js

```diff
@@ -33,4 +30,13 @@ describe('Card', function() {
   it('should have an id', function() {
     expect(card.id).to.equal(1);
   });
+
+  it('should store a different card', function() {
```

Comment:
Is this an appropriate way for testing that different arguments can be used for the class instance?